hknweb/people/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+
+class People(models.Model):
+  id                  = models.IntegerField(primary_key=True)
+  first_name          = models.CharField(max_length=255, null=False)
+  last_name           = models.CharField(max_length=255, null=False)
+  username            = models.CharField(max_length=255, null=False)
+  email               = models.CharField(max_length=255, null=False)
+  crypted_password    = models.CharField(max_length=255, null=False)
+  password_salt       = models.CharField(max_length=255, null=False)
+  persistence_token   = models.CharField(max_length=255, null=False)
+  single_access_token = models.CharField(max_length=255, null=False)
+  perishable_token    = models.CharField(max_length=255, null=False)
```

Comment:
Mentioned before, but we may choose a different signin authentication method (like Django's builtin one). Idk how but it's probably somewhere in the docs [here](https://docs.djangoproject.com/en/2.0/topics/auth/). Also, I think Michelle did a bit with that in pull request #3.

Reply:
Extending the default user: https://docs.djangoproject.com/en/2.0/topics/auth/customizing/#extending-django-s-default-user